### PR TITLE
Fix testing / installing on SLES 15

### PR DESCRIPTION
**Pull request by tas50**
Make sure we identify SLES 15 as suse for our zypper tests
Make sure we don't remove the symlinks on SLES when doing an upgrade or the new release will be broken

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Comment by tas50** [comment]
Apparently I'm too young to edit this file so calling on @btm and @lamont-granquist

File: omnibus/package-scripts/angrychef/postrm [hunk 1]
```diff
@@ -11,11 +11,19 @@ is_smartos() {
...
 
-is_darwin()
-{
+is_darwin() {
   uname -v | grep "^Darwin" 2>&1 >/dev/null
 }
 
+is_suse() {
+  if [ -f /etc/os-release ]; then
+    source /etc/os-release
```
Comment (btm) [inline omnibus/package-scripts/angrychef/postrm]: `source` is a bashism, you want `. /etc/os-release`

File: omnibus/package-scripts/angrychef/postrm [hunk 2]
```diff
@@ -11,11 +11,19 @@ is_smartos() {
...
+    [ $ID_LIKE == "sles" ] || [ $ID_LIKE == "suse" ]
```
Comment (btm) [inline omnibus/package-scripts/angrychef/postrm]: `==` is a bashism too. you want single `=`, i.e.
```
     [ $ID_LIKE = "sles" ] || [ $ID_LIKE = "suse" ]
```

Comment (btm) [reply omnibus/package-scripts/angrychef/postrm]: edit about the quotes around the variable --^

**Pull request merged**